egret/models/unit_commitment.py

```diff
@@ -652,6 +671,17 @@ def solve_unit_commitment(model_data,
                 flex_up_supp[dt] = value(m.FlexUpProvided[g,mt])
                 flex_dn_supp[dt] = value(m.FlexDnProvided[g,mt])
 
+            ## we can use pe.Constraint.slack() to calculate this...but we need some standard
+            ## constraint names in the unit commitment model library
+
+            ## pyomo doesn't add constraints that are skiped to the index set, so we also
+            ## need check here if the index exists.
+            slack_list = []
+            for constr in ramp_up_avail_constrs:
+                if (g,mt) in constr:
+                    slack_list.append(constr[g,mt].slack())
+
```

Comment:
A relative nit, but in the spirit of being Pythonic and minimizing code: lines 679-684 can / should probably be compacted into a single comprehension argument to min().